index.html

```diff
@@ -149,11 +149,11 @@ <h2 hidden>Game over</h2>
   // Display game over, cancel events
   function endGame(result) {
```

Comment:
```suggestion
<h2 hidden>Game over! Want to play again?! Just click refresh. 🧑‍🚀!</h2>
```
Let's make it a bit more friendly. 🤓